onnxruntime/core/providers/openvino/openvino_graph.cc

```diff
@@ -121,15 +125,9 @@ OpenVINOGraph::OpenVINOGraph(const onnxruntime::Node* fused_node) {
 
   // Create hardware specific OpenVINO network representation
   GetExecutableHandle(openvino_network_);
-
-
-
-  plugin_ = InferenceEngine::PluginDispatcher().getPluginByDevice(device_id_);
```

Comment:
Since the object member 'plugin_ ' is no longer used, its [definition](https://github.com/intel/onnxruntime/blob/openvino_R3/onnxruntime/core/providers/openvino/openvino_graph.h#L56) should should be cleaned up as well